modules/contentbox/modules/contentbox-admin/modules/contentbox-security/handlers/security.cfc

```diff
@@ -75,6 +75,16 @@ component extends="baseHandler"{
 				);
 			}
 
+			// If Global MFA is turned on and the user is not enrolled to a provider, then force it to enroll
+			if( twoFactorService.isForceTwoFactorAuth() AND !results.author.getIs2FactorAuth() ){
+				flash.put( "authorData", {
+					authorID 	= results.author.getAuthorID(),
+					rememberMe 	= rc.rememberMe,
+					securedURL  = rc._securedURL
+				} );
+				setNextEvent( "#prc.cbAdminEntryPoint#.authors.forceTwoFactorEnrollment" );
```

Comment:
I think this is the issue @gpickin . You should not force the enrollment by going to a secured author handler. The enforcement should stay within the two factor module. This cannot be.

Reply:
k